--- src/main/java/eu/hackathon/game/external/LevelData.java
+++ src/main/java/eu/hackathon/game/external/LevelData.java
@@ -12,5 +12,4 @@
 import java.io.IOException;
 import java.io.Serializable;
-import java.util.Iterator;
 
 @SuppressWarnings("unchecked")
@@ -57,6 +56,6 @@
 
                 index = 0;
-                for (Iterator iterator = data.iterator(); iterator.hasNext(); ) {
-                    this.data[index] = Integer.parseInt(Long.toString((Long) iterator.next()));
+                for(Object aData : data) {
+                    this.data[index] = Integer.parseInt(Long.toString((Long) aData));
                     index++;
                 }
@@ -65,6 +64,6 @@
 
                 index = 0;
-                for (Iterator iterator = interactables.iterator(); iterator.hasNext(); ) {
-                    this.entities[index] = Integer.parseInt(Long.toString((Long) iterator.next()));
+                for(Object interactable : interactables) {
+                    this.entities[index] = Integer.parseInt(Long.toString((Long) interactable));
                     index++;
                 }
