--- src/main/java/eu/hackathon/cheesecake/internal/graphics/ImageType.java
+++ src/main/java/eu/hackathon/cheesecake/internal/graphics/ImageType.java
@@ -6,5 +6,5 @@
 /**
  * This enum defines all available image types.
- * <p>
+ * 
  * <p>This enum is required to load images of a specific type.</p>
  *
@@ -43,5 +43,5 @@
      * Returns the format (package-private).
      *
-     * @Return String The required format.
+     * @return String The required format.
      */
     String format() {
