--- src/main/java/eu/hackathon/cheesecake/external/ui/GUI.java
+++ src/main/java/eu/hackathon/cheesecake/external/ui/GUI.java
@@ -26,5 +26,5 @@
 
     public static boolean drawButton(ExtendedGraphics extendedGraphics, Image image, int x, int y, int width, int height) {
-        extendedGraphics.drawImage(image, x, y);
+        extendedGraphics.drawImage(image, x, y, width, height);
         return Mouse.isInArea(x, y, width, height) && Mouse.getKeyUp(KeyCode.MOUSE_LEFT);
     }
